articles/code_coverage.dd

```diff
@@ -35,12 +35,12 @@ $(LI It can be used to track down why a particular section of code
 $(LI Since execution counts are given for each line, it is possible
      to use the coverage analysis to reorder the basic blocks in
      a function to minimize jmps in the most used path, thus
-     optimizing it.)
-)
+     optimizing it. This is known as $(LINK2 https://en.wikipedia.org/wiki/Profile-guided_optimization, profile-guided optimization)
```

Comment:
Your addition is fine, but the existing sentence is in need of revision. That 'thus optimizing it' at the end is disruptive. And it's I suggest removing it and rewriting as: "Since execution counts are given for each line, a possible optimization is to use the coverage analysis to reorder the basic blocks in a function to minimize jmps in the most used path."

Reply:
OK will do (if you're able to push to my branch from your end feel free since I'm in bed). Minor point (subject of my blog post coming ASAP): This is one of the only optimization you can really act blindly on - i.e. the branch predictor is very good but needs a hand for that last few percent.

Reply:
I've reworded it.